contracts/util/AssetSettingsLib.sol

```diff
@@ -14,12 +15,13 @@ library AssetSettingsLib {
 
     /**
         @notice This struct manages the asset settings in the platform.
-        @param cTokenAddress cToken address associated to the asset. 
+        @param tokenAddresses the token addresses associated to the asset. 
         @param maxLoanAmount max loan amount configured for the asset.
      */
     struct AssetSettings {
-        // It represents the cTokenAddress or 0x0.
-        address cTokenAddress;
+        // It represents a mapping of the token name and the tokenAddress or 0x0.
+        // address tokenAddress;
+        mapping(bytes32 => address) tokenAddresses;
```

Comment:
Remember this struct cannot be returned in a public/external function due to the new mapping field.